pennylane_lightning/_serialize.py

```diff
@@ -113,11 +113,11 @@ def _serialize_obs(tape: QuantumTape, wires_map: dict, use_csingle: bool = False
             if is_tensor:
                 for o_ in o.obs:
                     if not _obs_has_kernel(o_):
-                        params.append(o_.matrix.ravel().astype(ctype))
+                        params.append(o_.get_matrix().ravel().astype(ctype))
```

Comment:
@maliasadi no action needed for now, but this is a temporary name that still might change before the next release - I'll update you once I know the final decision!

Reply:
Should we go ahead with merging this, or is it best to await on the updated name?

Reply:
I think that's better to wait until the op refactoring of the coming release gets settled.

Reply:
There are definitely going to be more changes and so we can address all within this PR.

Reply:
Hi @josh146. Would it be any further updates on ops requiring changes/fixes in Lightning soon? I want to know when is suitable for this PR to be merged.

Reply:
Hey @maliasadi, you can update this now to use `qml.matrix(o_)` 🙂 then it should be good to merge!